### PR TITLE
DRY Schedule section 

### PR DESCRIPTION
**Pull request by 2root2**
Added `data/schedule.json` and `js/schedule.js `to dynamically generate Schedule section
Have a look at my [gh-pages](https://2root2.github.io/inpycon2018/)
Note: Section is currently commented on `master` branch
Fixes : #165 
@ananyo2012 Please Review.

### CODE REVIEW

**Comment by 2root2** [comment]
@ananyo2012 Please review requested changes.

**Comment by ananyo2012** [comment]
Ok since we are yet to get the schedule from the CFP team can you remove the Schedules for all days except the first day from the json file. I want to see how it looks with only one day.

**Comment by 2root2** [comment]
@ananyo2012 Have a look at my [gh-pages](https://2root2.github.io/inpycon2018/) for requested changes. 
Note: It shows empty block for other days, need to modify javascript to display something like 
`Coming Soon..`.

**Comment by ananyo2012** [comment]
I think you removed just the schedule. Remove the entire entries for the
next 4 days. Also can you remove Breakfast from "Registration and
Breakfast" ? There will be no Breakfast.

**Comment by 2root2** [comment]
@ananyo2012  check here [gh-pages](https://2root2.github.io/inpycon2018/)
Note: It breaks navigation from date menu, that's why I had kept data in `json` file.

**Comment by ananyo2012** [comment]
@2root2 The easiest solution to this would be to comment the schedule nabvar for  the days for which data is not available.

**Comment by 2root2** [comment]
@ananyo2012 Done check [gh-pages](https://2root2.github.io/inpycon2018/)

**Comment by ananyo2012** [comment]
@2root2 The workshop schedule is finalized now. Can you update the PR with the schedule and uncomment the schedule now. I will merge this once its added. Here is the schedule

workshop-schedule

**Comment by 2root2** [comment]
@ananyo2012 Don't we want Speaker for all the workshops?

**Comment by ananyo2012** [comment]
Don't worry about the speakers. I will add them later. Keep them blank for
now.

**Comment by 2root2** [comment]
@ananyo2012 Added given data check [gh-pages](
https://2root2.github.io/inpycon2018/)

**Comment by ananyo2012** [comment]
Thanks a lot for hardwork. I will merge this shortly after I get an approval from the CFP team.

File: js/schedule.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                                    item.Tracks[i].Title+
+                                "</a>"+
+                            "</strong>"+
+                            "<br><em>By "+item.Tracks[i].Speaker+"</em>"+
+                            "<br><span class='badge'>"+item.Tracks[i].Hall+"</span>"+
+                        "</div>";
+    }
+    rowContent += "</div>";
+    return rowContent;
+}
```
[the remark this answers is not in the document]
Comment (ananyo2012) [reply js/schedule.js]: Fixed

**Comment by ananyo2012** [comment]
Everything seems fine. Merging this.

**Pull request merged**